### PR TITLE
framework/media: Add MediaPlayer getId api

### PR DESCRIPTION
**Pull request by junmin-kim**
Add new player API: getId() to get player ID.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [ef6d842ef661601a24ba2a7f95a05928a8e76d97] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [ef6d842ef661601a24ba2a7f95a05928a8e76d97] - Code Rule Check Result:
framework/include/media/MediaPlayer.h:175: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:175: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:175: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:175: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:175: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:175: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/include/media/MediaPlayer.h:186: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:186: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:186: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:186: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/include/media/MediaPlayer.h:189: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:189: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:189: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:51: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/src/media/MediaPlayerImpl.h:51: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:51: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:51: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:51: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
framework/src/media/MediaPlayerImpl.h:69: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:69: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:69: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:69: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:69: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:69: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/src/media/MediaPlayerImpl.h:70: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:70: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:70: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:70: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/src/media/MediaPlayerImpl.h:89: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:89: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:89: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:89: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:90: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:90: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:90: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:90: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:90: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:90: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:90: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:93: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:93: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:93: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:96: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:97: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:98: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:98: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:98: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:99: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:99: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:99: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:99: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:99: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)

File: framework/src/media/MediaPlayerImpl.cpp [hunk 1]
```diff
@@ -366,6 +366,11 @@ void MediaPlayerImpl::pausePlayer()
 	mCurState = PLAYER_STATE_PAUSED;
 }
 
+int MediaPlayerImpl::getId()
+{
+	return mId;
```
Comment (davidfather) [inline framework/src/media/MediaPlayerImpl.cpp]: Any possibility of an integer overflow?

Comment (jsdosa) [reply framework/src/media/MediaPlayerImpl.cpp]: @davidfather yes, you right.
so, our framework will delete id.
could you check the pr : https://github.com/Samsung/TizenRT/pull/2034

Comment (jsdosa) [reply framework/src/media/MediaPlayerImpl.cpp]: @junmin-kim do not merge!!

**Pull request closed without merging**